aten/src/ATen/native/cuda/UpSampleTrilinear3d.cu

```diff
@@ -325,21 +344,27 @@ static void upsample_trilinear3d_backward_out_cuda_template(
 
   grad_input.resize_(
       {nbatch, channels, input_depth, input_height, input_width});
+  // A contiguous tensor is required for the kernel launch config
+  grad_input.contiguous();
+  // initialization to zero is required here. As we launch one thread per output
+  // element, and atomicAdd to input gradient. Given a sparse sampling case, our
+  // threads are not covering the whole input tensor.
```

Comment:
it's not only that you are not covering the whole input tensor, even elements you cover are adding to existing values, if you initialize with garbage, result will be garbage too, so a bit clearer wording in the comment